src/renderers/WebGLRenderTarget.js

```diff
@@ -49,9 +49,12 @@ THREE.WebGLRenderTarget.prototype = {
 		}
 	},
 
-	clone: function () {
+	clone: function (tmp) {
```

Comment:
We need to discuss if `clone()` shoud take an argument. I do not think it should -- it becomes too much like `copy()`. /ping @mrdoob